pyk/src/pyk/cli_utils.py

```diff
@@ -36,25 +39,38 @@ def file_path(s: str) -> Path:
 
 def run_process(
     args: Union[str, Sequence[str]],
-    logger: Logger,
     *,
-    log_level: int = logging.DEBUG,
+    check: bool = True,
+    profile: bool = False,
+    suppress_stderr: bool = False,
```

Comment:
How do you feel about doing this instead:
```suggestion
    stderr = subprocess.PIPE,
```

This would give us the flexability to capture both stdout and stderr, or whatever else the user wants.

Reply:
No, I want users to have to turn off stderr intentionally, _not_ to have it off by default. We should never be silently ignoring warnings from any software we're calling. If a piece of our software is being too annoying with warnings, we should fix either the source of the warnings or change our software to be quieter. If some other piece of software is giving annoying warnings, we can still turn it off.